### ISSUE TITLE
Hotkeys ignored when a GPIW is in focus

### ISSUE THREAD

**Issue by riverwanderer** [vassalengine/vassal#11170]
Vassal v3.6.4.

Player ignores hotkeys when a GPIW is in focus.

To replicate, open a module with a GPIW and a simple hotkey such as a show charts window or similar.
Open a GPIW
Immediately try the hotkey. It should fail to respond.
Click away from the GPIW and re-try the hotkey; it will then work.
Rinse and repeat.

**Comment by uckelman** [comment]
Does this happen with 3.5.8?

**Comment by riverwanderer** [comment]
Yes; it occurs as far back as 3.2.17. A similar issue used to affect hotkeys and palettes but that was fixed in v3.5.

**Comment by riverwanderer** [comment]
Test in v3.5.8 and it occurs there also. The problem for palettes arises when a piece has been selected in the palette window. Simply opening the window and pressing the hotkey does work, as does selecting a tab in the palette and then pressing the hotkey. As soon as a piece has been selected, the hotkey does not work.
